Tangem/Modules/StakingDetails/StakingDetailsViewModel.swift

```diff
@@ -316,6 +316,17 @@ private extension StakingDetailsViewModel {
         }
     }
 
+    private func openFlow(for action: StakingAction) {
+        switch action.type {
+        case .stake:
+            coordinator?.openStakingFlow()
+        case .pending(let pendingAction) where pendingAction.isVoteLocked:
```

Comment:
вот так вообще круто будет )

```suggestion
        case .pending(.voteLocked):
```

Reply:
поправил